flink/src/test/scala/com/metamx/tranquility/test/FlinkDruidTest.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.metamx.tranquility.test
+
+import java.util.concurrent.TimeUnit
+
+import com.metamx.common.scala.Logging
```

Comment:
Could you please import the codestyle jar at https://github.com/druid-io/druid/blob/master/intellij_formatting.jar and optimize the imports? That should split up the grouped `{a, b, c}` imports, which is the style we use for Scala stuff in Druid projects.